src/services/study/network.js

```diff
@@ -192,13 +192,13 @@ export function fetchNetworkElementInfos(
     }
     urlSearchParams.append('elementType', elementType);
     urlSearchParams.append('infoType', infoType);
-    const additionalParams = new Map();
+    const optionalParams = new Map();
```

Comment:
Personnaly I prefered "additionalParams". 'inUpstreamBuiltParentNode' by example is an optional param but is not inside `optionalParameters'. It is a contradiction.

But it is not a big deal.

Reply:
name choosen by TL :)